libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -76,6 +78,7 @@ export class CounterfactualList extends React.Component<
     super(props);
     this.state = {
       data: {},
+      displayedData: {},
```

Comment:
Storing the data twice could eventually lead to inconsistencies if we're not extremely careful. I'm wondering if it's possible to manipulate the data before displaying rather than storing it twice. Did you consider that option? If so, I'm curious why that might not work.

Reply:
Removed the change so that we only store the data once. The reason we can remove this is because, we don't need to store a numeric version of data here. We can just pass the string version with decimal point, and it is converted to number before sending the request (in setCustomRowProperty method inside CounterfactualChart.tsx where we have `const asNumber = +newValue`)